cmd/main.go

```diff
@@ -205,6 +206,19 @@ func maxDuration(dur time.Duration, durations ...time.Duration) time.Duration {
 	return ret
 }
 
+func importLocalCluster(ctrlMgr manager.Manager, log *logrus.Logger) {
+	// Splitting into API reader and writer interfaces as the reader bypasses the cache settings that are on the regular client
+	// and we need to be able to read secrets that we cannot read with the client due to how the cache is configured.
+	// The cachedApiClient can be used for writes as these are unaffected by cache.
+	localClusterImportOperations := localclusterimport.NewLocalClusterImportOperations(ctrlMgr.GetAPIReader(), ctrlMgr.GetClient(), log)
+	localClusterImport := localclusterimport.NewLocalClusterImport(&localClusterImportOperations, log)
+	err := localClusterImport.ImportLocalCluster()
+	if err != nil {
+		// Failure to import the local cluster is not fatal but we should warn in the log.
+		log.Warnf("Could not import local cluster into ACM due to error %s", err.Error())
```

Comment:
```suggestion
		log.Warnf("Could not import local cluster due to error %s", err.Error())
```

Reply:
I don't it should mention ACM specifically:
1. It might not be relevant to some deployment types we support.
2. ACM is a downstream name.

Reply:
What is the correct terminology for what we are doing here?

Reply:
After discussion with @filanov we have settled on "import local cluster" as the terminology to be used.

Reply:
+1 to what Nir suggested, we have both ACM and MCE, so his suggestion cover both